### ISSUE TITLE
Fix v14.3.1 CMake build

### ISSUE THREAD

**Issue by joewallwork** [fetch4/GISS-GC#2]
Currently, building GISS-GC on branch `dev/giss-gc-14` gets 92% of the way through the GEOS-Chem CMake build before crashing with
```
Error copying Fortran module "mod/get_met_mod.mod".  Tried "mod/GET_MET_MOD.mod" and "mod/get_met_mod.mod".
```

This will need fixing in order to progress with the version upgrade.

**Comment by joewallwork** [comment]
See also https://github.com/fetch4/geos-chem/issues/2.

**Comment by joewallwork** [comment]
Closed by #3.
